### ISSUE TITLE
Homebridge beta 1.3.0-beta.67: characteristic was supplied illegal value: null!

### ISSUE THREAD

**Issue by dzeleski** [chrisjshull/homebridge-nest#369]
**Describe the bug**
Install beta 67 warnings occur due to illegal values

```
[18/02/2021, 10:50:32] [Nest] initing home_away_sensor "Home Occupied": deviceId: <REMOVED> structureId: <REMOVED>
[Home Occupied@@Firmware Revision] characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics
[18/02/2021, 10:50:32] [Nest] initing temp_sensor (P) "Bedroom": deviceId: <REMOVED> structureId: <REMOVED>
[Bedroom@@Firmware Revision] characteristic was supplied illegal value: undefined! This might throw errors in the future!
```

Environment

    OS: Raspbian
    Node.js: v14.15.5
    NPM: v6.14.11
    homebridge-ring: v9.15.1
    homebridge: v1.3.0-beta.67

**Comment by cscott365** [comment]
Same as issue #368 submitted just a bit before you.

**Comment by adriancable** [comment]
Should be fixed in 4.5.2 - please let me know if not the case.
